Makefile

```diff
@@ -211,74 +202,6 @@ delete-tag:
 docker-otelcol:
```

Comment:
Doesn't this target require the binary that we are not building anymore?

Reply:
Removed